tuxemon/core/states/world/worldstate.py

```diff
@@ -588,7 +588,7 @@ def get_explicit_tile_exits(self, position, tile, skip_nodes):
         # does the tile explicitly define exits?
         try:
             adjacent_tiles = list()
-            for direction in tile["exit"]:
+            for direction in tile['exit']:
```

Comment:
Do not change the quotes.  Double quotes is part of the PEP-8 style guidelines, which we should try to conform to.  Eventually, we will follow the "Black" standard, which will force all to double quotes.  Please revert this.

Reply:
Fixed in the last commit. I noticed that single and double quotes were both used in different places, and since single quotes are most common I switched to that thinking it's more aesthetic. I asked about this on Discord and was clarified that double quotes are the way to go, so I reverted this and also fixed a few similar cases further down in the same code area.